roles/KubevirtTemplateValidator/templates/service.yaml.j2

```diff
@@ -58,6 +58,9 @@ spec:
       labels:
         kubevirt.io: virt-template-validator
     spec:
+      affinity: {{ cr_info['spec']['affinity'] | default("", true) | from_yaml }}
+      nodeSelector: {{ cr_info['spec']['nodeSelector'] | default("", true) | from_yaml }}
```

Comment:
This looks like assigning of incompatible types, does the code work?

The `nodeSelector` field of `core.v1.PodSpec` struct has type `map[string]string`, but the `nodeSelector` in `TemplateValidatorSpec` is `v1.NodeSelector`.

Reply:
The code works, but I will take another look at the types maybe I missed something

Reply:
You are correct, I will change that